servicemanager/servicemanager.go

```diff
@@ -215,6 +229,33 @@ func (sm *ServiceManager) doInstallService(newService ServiceInfo, imageURL stri
 	return nil
 }
 
+func (sm *ServiceManager) doApplyServise(service ServiceInfo) (err error) {
+	oldServices, err := sm.serviceInfoProvider.GetAllServiceVersions(service.ServiceID)
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+
+	for _, oldService := range oldServices {
+		if oldService.AosVersion == service.AosVersion {
+			continue
+		}
+
+		if err := os.RemoveAll(oldService.ImagePath); err != nil {
+			log.Errorf("Can't remove old service: %s", err)
+		}
+
+		if err := sm.serviceInfoProvider.RemoveService(oldService); err != nil {
+			log.Errorf("Can't remove old service from DB: %s", err)
```

Comment:
Do not mention DB. `Can't remove from storage`.

Reply:
fixed